app/src/main/res/layout/main_nav_header.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <import type="org.fossasia.openevent.app.common.utils.core.DateUtils" />
+
+        <variable
+            name="event"
+            type="org.fossasia.openevent.app.common.data.models.Event" />
+    </data>
+
+    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/nav_header_height">
+
+        <ImageView
```

Comment:
set `android:scaleType` to `centerCrop` or `fitCenter` to ensure an image always fits to the header.